content/doc/tutorials/create-a-pipeline-in-blue-ocean.adoc

```diff
@@ -8,430 +8,294 @@ section: doc
+
+. Select the *X* at the top-right to return to the main Blue Ocean interface.
 
 
 === Add a final deliver stage to your Pipeline
 
-. From the main Blue Ocean interface, click *Branches* at the top-right to
-  access your repository's `master` branch.
-. Click the `master` branch's "Edit Pipeline" icon
-  image:tutorials/pipeline-in-blueocean-21a-edit-pipeline-on-branch.png[alt="Edit
-  Pipeline on branch",width=3%]
-  to open the Pipeline editor for this branch.
-. In the main Pipeline editor, click the *+* icon to the right of the *Test*
-  stage you created <<add-a-test-stage-to-your-pipeline,above>> to open the new
-  stage panel. +
+. From the main Blue Ocean interface, select *Branches* at the top-right to access your repository's `master` branch.
+. Select the `master` branch's "Edit Pipeline" icon image:tutorials/pipeline-in-blueocean-21a-edit-pipeline-on-branch.png[alt="Edit Pipeline on branch",width=3%] to open the Pipeline editor for this branch.
+. In the main Pipeline editor, select the *+* icon to the right of the *Test* stage you created <<add-a-test-stage-to-your-pipeline,above>> to open the new stage panel.
++
 [.boxshadow]
-image:tutorials/pipeline-in-blueocean-31-add-deliver-stage.png[alt="Add 'Deliver'
-stage",width=60%]
-. In this panel, type `Deliver` in the *Name your stage* field and then click
-  the *Add Step* button below to open the *Choose step type* panel.
-. In this panel, click *Shell Script* near the top of the list.
-. In the resulting *Deliver / Shell Script* panel, specify
-  `./jenkins/scripts/deliver.sh` and then click the top-left back arrow icon
-  image:tutorials/pipeline-in-blueocean-16a-return-from-step-icon.png[alt="Return
-  from step icon",width=3%] to return to the Pipeline stage editor.
+image:tutorials/pipeline-in-blueocean-31-add-deliver-stage.png[alt="Add 'Deliver' stage",width=60%]
+. In this panel, type `Deliver` in the *Name your stage* field and then select *Add Step* below to open the *Choose step type* panel.
+. In this panel, select *Shell Script* near the top of the list.
+. In the resulting *Deliver / Shell Script* panel, specify `./jenkins/scripts/deliver.sh` and then select the top-left back arrow icon image:tutorials/pipeline-in-blueocean-16a-return-from-step-icon.png[alt="Return from step icon",width=3%] to return to the Pipeline stage editor.
++
 [.boxshadow]
-image:tutorials/pipeline-in-blueocean-32-add-next-step.png[alt="Add next
-step",width=50%] +
-  *Note:* For an explanation of this step, refer to the `deliver.sh` file itself
-  located in the `jenkins/scripts` of your forked repository on GitHub.
-. Click the *Add Step* button again.
-. In the *Choose step type* panel, type `input` into the *Find steps by name*
-  field.
+image:tutorials/pipeline-in-blueocean-32-add-next-step.png[alt="Add next step",width=50%]
++
+*Note:* For an explanation of this step, refer to the `deliver.sh` file itself located in the `jenkins/scripts` of your forked repository on GitHub.
```

Comment:
Our notes also have different formatting. Not sure if you want to make your notes like CloudBees docs. See this page for examples: https://docs.beescloud.com/docs/beewrite/latest/syntax/admonitions